src/components/donation-flow/alerts/AlertsColumn.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { AlertProps, Box, List, ListItem, ListItemText, SxProps, Typography } from '@mui/material'
+import { useFormikContext } from 'formik'
+import { AnchoredAlert } from './AnchoredAlert'
+import {
+  DonationFormDataAuthState,
+  DonationFormDataPaymentOption,
+  DonationFormDataV2,
+} from '../helpers/types'
+import { useElements } from '@stripe/react-stripe-js'
+
+const cardAlertDescription = `Таксата на Stripe се изчислява според района на картодържателя: 1.2% + 0.5лв. за Европейската икономическа зона`
+const bankAlertDescription = `Таксата за транзакция при банков превод зависи от индивидуалните условия на Вашата банка. от (0-4лв)`
+
+const paymentMethodAlertMap = {
+  [DonationFormDataPaymentOption.CARD]: cardAlertDescription,
+  [DonationFormDataPaymentOption.BANK]: bankAlertDescription,
+}
+
+function AlertsColumn({
+  sectionsRefArray,
+}: {
+  sectionsRefArray: React.MutableRefObject<HTMLDivElement | null>[]
+}) {
+  const {
+    values: { payment, authentication },
+  } = useFormikContext<DonationFormDataV2>()
+
+  const liSx: SxProps = {
+    py: 0,
+  }
+  const [updatedRefArray, setUpdatedRefArray] =
+    React.useState<React.MutableRefObject<HTMLDivElement | null>[]>(sectionsRefArray)
+  const elements = useElements()
+  const paymentElement = elements?.getElement('payment')
+  paymentElement?.once('ready', () => {
+    setUpdatedRefArray([...sectionsRefArray])
+  })
+  const alerts: { [key: string]: AlertProps } = {
+    'select-payment-method': {
+      color: 'info',
+      children: <Typography>{payment && paymentMethodAlertMap[payment]}</Typography>,
+      icon: false,
+      sx: {
+        display: payment ? 'flex' : 'none',
+      },
+    },
+    'select-authentication': {
+      color: 'info',
+      children: (
+        <Box>
+          <Typography>Избирайки да се впишете. ще можете да:</Typography>
+          <List
```

Comment:
We can extract this list as separate component to simplify the rendering of alerts

Reply:
We might also expand the text and reduce the right padding/margin to contain more chars on single row

![image](https://user-images.githubusercontent.com/893608/216359587-568614f9-60e5-458f-990e-8acb17b8ca30.png)

Reply:
Removed all on the list items, but kept the one on the `X` button row